crypto/include/internal/cryptlib_int.h

```diff
@@ -7,17 +7,16 @@
  * https://www.openssl.org/source/license.html
  */
 
+#include <openssl/core.h>
 #include "internal/cryptlib.h"
 
 /* This file is not scanned by mkdef.pl, whereas cryptlib.h is */
 
-struct thread_local_inits_st {
-    int async;
-    int err_state;
-    int rand;
-};
-
-int ossl_init_thread_start(uint64_t opts);
+int ossl_init_thread_start(void *arg,
+                           OSSL_thread_stop_handler_fn handfn);
+int init_thread(void);
+void cleanup_thread(void);
+void ossl_ctx_thread_stop(void *arg);
```

Comment:
These start/stop names look different

Reply:
Not sure I understand your point. There isn't a "start" equivalent of `ossl_ctx_thread_stop`

Reply:
What about `ossl_init_thread_start`?

Reply:
The `stop` equivalent of `ossl_init_thread_start` is `ossl_init_thread_stop`. There is no "start" equivalent of `ossl_ctx_thread_stop`.